lib/motorcontrollers/controller.cpp

```diff
@@ -3,5 +3,73 @@
 namespace hyped::motorcontrollers {
```

Comment:
i don't like the namespace name

motors?

Reply:
Isn't this an issue for a different pr? And is motors actually better?
I chose motorcontrollers as it:
a) directly attaches the namespace to us, as all the other software teams do, 
and b) because the code is for interacting with the controllers, not directly the motors. Though I recognise this point is kinda debatable at best

Reply:
I don't like `motorcontrollers` because it's (a) too long. That's it. Also, of course the job of the software is to control the motors (just like sensors controls the sensors). This involves talking to the dedicated controllers as well as things like the STN board (which I suppose is a controller in some sense?)

I agree that this is the wrong PR, it's just the first time I see it.

Reply:
#16